services/root.js

```diff
@@ -1,7 +1,23 @@
-import { generateApiClient } from "@app/utils/apiUtils";
-const itunesApi = generateApiClient("itunes");
+// import { generateApiClient } from "@app/utils/apiUtils";
+// const itunesApi = generateApiClient("itunes");
 
-export const getRecommendations = async () => {
-  const res = await itunesApi.get(`search?term=`);
-  console.log("res", res);
-};
+// export const getRecommendations = async () => {
```

Comment:
Keeping this code commented for now since I'll needing this one once I work on recommendations feature later